src/StringExtensions/OwOConverter.cs

```diff
@@ -8,17 +8,20 @@ public static class OwOConverter
     {
         private static readonly Random RandomInt = new Random((int)DateTime.UtcNow.Ticks);
         private static readonly List<string> Faces = new List<string> { ";w;", "owo", "UwU", ">w<", "^w^", "*w*" };//"(`・ω・´)(・`ω´・)" doesn't encode right :C
-        private static readonly MatchEvaluator FaceEvaluator = _ => " " + Faces[RandomInt.Next(Faces.Count)] + " ";
+        private static readonly MatchEvaluator FaceEvaluator = _ => $" {Faces[RandomInt.Next(Faces.Count)]} ";
+        private static readonly IDictionary<string, string> map = new Dictionary<string, string>{
+            {@"(r|l)", "w"},
+            {@"(R|L)", "W"},
+            {"n([aeiou])", @"ny$1"},
+            {"N([aeiou])", @"Ny$1"},
+            {"N([AEIOU])", @"Ny$1"},
+            {@"(ove)", "uv"},
+            {@"(ou)", "ew"}
+        };
+        private static readonly Regex regex = new Regex(String.Join("|",map.Keys));
         public static string ConvertToOwO(this string input)
         {
-
-            input = Regex.Replace(input, @"(r|l)", "w", RegexOptions.Multiline);
-            input = Regex.Replace(input, @"(R|L)", "W", RegexOptions.Multiline);
-            input = Regex.Replace(input, "n([aeiou])", @"ny$1", RegexOptions.Multiline);
-            input = Regex.Replace(input, "N([aeiou])", @"Ny$1", RegexOptions.Multiline);
-            input = Regex.Replace(input, "N([AEIOU])", @"Ny$1", RegexOptions.Multiline);
-            input = Regex.Replace(input, @"(ove)", "uv", RegexOptions.Multiline);
-            input = Regex.Replace(input, @"(ou)", "ew", RegexOptions.Multiline);
+            input = regex.Replace(str, m => map[m.Value], RegexOptions.Multiline);
```

Comment:
```suggestion
            input = regex.Replace(input, m => map[m.Value], RegexOptions.Multiline);
```
Use the local variables v_v